### PR TITLE
En 865 data types endpoints

### PR DESCRIPTION
**Pull request by eugeniosant**
Note: the task has to be complete

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: engine/src/main/java/org/entando/entando/aps/system/services/entity/EntityManagerService.java [hunk 1]
```diff
@@ -13,73 +13,95 @@
...
 
-    @Autowired
-    private List<IEntityManager> entityManagers;
+    @Override
+    protected IDtoBuilder<IApsEntity, EntityTypeFullDto> getEntityTypeFullDtoBuilder(IEntityManager masterManager) {
+        return new EntityTypeFullDtoBuilder(masterManager.getAttributeRoles());
+    }
+
+    //@Autowired
+    //private List<IEntityManager> entityManagers;
```
Comment (joewhite101) [inline engine/src/main/java/org/entando/entando/aps/system/services/entity/EntityManagerService.java]: remove

File: engine/src/main/java/org/entando/entando/aps/system/services/entity/EntityManagerService.java [hunk 2]
```diff
@@ -13,73 +13,95 @@
...
 
-    private IDtoBuilder<IEntityManager, EntityManagerDto> dtoBuilder;
+    @Override
+    public PagedMetadata<EntityTypeShortDto> getShortEntityTypes(String entityManagerCode, RestListRequest requestList) {
+        return super.getShortEntityTypes(entityManagerCode, requestList);
+    }
 
-    protected List<IEntityManager> getEntityManagers() {
-        return entityManagers;
+    protected Map<String, String> getEntityManagerFieldNameMapping() {
```
Comment (joewhite101) [inline engine/src/main/java/org/entando/entando/aps/system/services/entity/EntityManagerService.java]: Does this need to be a method?

Comment (eugeniosant) [reply engine/src/main/java/org/entando/entando/aps/system/services/entity/EntityManagerService.java]: Removed!

File: engine/src/main/java/org/entando/entando/aps/system/services/entity/EntityManagerService.java (hunk 1 above)
Comment (eugeniosant) [reply engine/src/main/java/org/entando/entando/aps/system/services/entity/EntityManagerService.java]: Removed!

**Pull request merged**